src/pages/Home/components/HomeHeader/styles.scss

```diff
@@ -1,30 +1,49 @@
 .Header {
   background-color: $secondary;
   border-radius: 5px;
-  padding: 1.95rem 1.25rem 1.3rem 1.95rem;
+  padding: 1.95rem 2.375rem 2.5rem 2.375rem;
+  min-height: 245px;
   display: flex;
   justify-content: space-between;
 
   &__calendar {
+    border-radius: 5px;
+    padding: 0.32rem 0.625rem;
+
     .q-btn__content {
-      padding: 0.1rem 0.5rem;
+      padding: 0px;
```

Comment:
Maybe px is not needed here

Reply:
Yes, isn't needed. Will remove. Thanks!

Reply:
Seems like it was not removed, please check

Reply:
I confused him with another, now is removed, please check